gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/scheduled/EHRTimeoutHandlerTest.java

```diff
@@ -1,8 +1,6 @@
 package uk.nhs.adaptors.pss.translator.task.scheduled;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
Try not to do import *, do exact import. Possibly just copy and paste what was there before